.github/workflows/ci.yml

```diff
@@ -37,19 +39,21 @@ jobs:
       - name: Lint
         run: npm run lint
 
-      - name: Test
-        if: github.ref != 'refs/heads/master'
-        run: npm run test
-
-      - name: Test & Coverage
-        if: github.ref == 'refs/heads/master'
+      - name: Tests & Coverage
         run: |
           npm run test
-          export COVERALLS_REPO_TOKEN=${{ secrets.COVERALLS_REPO_TOKEN }}
           npm run coverage
         env:
           CI: true
 
+      - name: Sonar Code Analysis
+        if: "github.repository == 'pioardi/poolifier'"
+        run: |
+          npm run sonar
```

Comment:
Just a minor, but these two lines can be merged into one `run: npm run sonar`

Reply:
I would prefer to keep as is for further easy troubleshooting without any impact